packages/account/src/Sections/Security/Passwords/deriv-password.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon, Text } from '@deriv/components';
+import { toTitleCase } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services/ws-methods';
+import FormSubHeader from 'Components/form-sub-header';
+import ChangePasswordForm from './change-password-form.jsx';
+import PasswordsStatic from './passwords-static.jsx';
+import SentEmailModal from './sent-email-modal.jsx';
+
+const DerivPassword = ({ email, is_social_signup, social_identity_provider }) => {
+    const [is_unlink_modal_open, setIsUnlinkModalOpen] = React.useState(false);
+    const [is_sent_email_modal_open, setIsSentEmailModalOpen] = React.useState(false);
+
+    const onClickSendEmail = () => {
+        WS.verifyEmail(email, 'reset_password');
+        setIsUnlinkModalOpen(false);
+    };
+
+    const capitalize_identifier = social_identity_provider ? toTitleCase(social_identity_provider) : '';
+
+    const getSocialidentityProvider = () => {
```

Comment:
Nit, but this is unnecessary as it's only used once, why not add it as a conditional child (line 47ish) ? 🧒